autoscoper/src/ui/AutoscoperMainWindow.cpp

```diff
@@ -2438,9 +2438,53 @@ void AutoscoperMainWindow::key_k_pressed()
 {
   ui->toolButtonNextCurveSet->click();
 }
-/*void AutoscoperMainWindow::key_p_pressed(){
-   ui->toolButtonRetrack->click();
-   } */
+void AutoscoperMainWindow::key_p_pressed()
+{
+  // Get the current position of the manipulator and the volume
+  Mat4d manip = getManipulator(-1)->transform();
+  CoordFrame vol_world = (CoordFrame::from_matrix(trans(manip)) * *tracker->trial()->getVolumeMatrix(-1));
+
+  // The translation center is the relative position of the center of the volume to the corner coordinate.
```

Comment:
Could you confirm that "translation center" is synonym with "translation origin" ?

Reply:
Yes this is correct, I believe this was called the "translation center" since, by default, it is set to the center of the volume.